common/djangoapps/track/views/__init__.py

```diff
@@ -78,6 +81,36 @@ def user_track(request):
 
     return HttpResponse('success')
 
+perflog = logging.getLogger("perflog")
+
+
+def performance_log(request):
+    """
+    Log when POST call to "performance" URL is made by a user.
+    Request should provide "event" and "page" arguments.
```

Comment:
Does this need to be enforced? Tried/caught if those arguments are not in the request?